miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppView.kt

```diff
@@ -0,0 +1,70 @@
+package com.rakuten.tech.mobile.miniapp.view
+
+import android.content.Context
+import android.view.View
+import com.rakuten.tech.mobile.miniapp.MiniAppInfo
+import com.rakuten.tech.mobile.miniapp.sdkExceptionForInvalidArguments
+
+/**
+ * This class can be use to create miniapp instances.
+ * @param context of where the miniapp is loaded.
+ * @param config provide the necessary configuration.
+ */
+class MiniAppView(val context: Context, val config: MiniAppConfig) {
+
+    private var miniAppViewHandler: MiniAppViewHandler =
+        config.miniAppSdkConfig?.let { MiniAppViewHandler(context, it) }!!
+
+    /**
+     * Creates a mini app.
+     * The mini app is downloaded, saved and provides a view when successful.
+     * @param appId mini app id.
+     * @param fromCache allow host app to load miniapp from cache.
+     * @throws [MiniAppNotFoundException] when the specified project ID does not have any mini app exist on the server.
+     * @throws [MiniAppHasNoPublishedVersionException] when the specified mini app ID exists on the
+     * server but has no published versions
+     * @throws [MiniAppSdkException] when there is any other issue during fetching,
+     * downloading or creating the view.
+     * @throws [RequiredPermissionsNotGrantedException] when the required permissions of the manifest are not granted.
+     */
+    suspend fun create(miniAppId: String, fromCache: Boolean = false): View? = when {
```

Comment:
@munir-rakuten can you rename this to `load`? naming should be aligned

Reply:
Sure, done